ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/Util/Orphans/SignableRepresentation.hs

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
We always use `-Wno-orphans`

Reply:
So you would rather not have this here? My inclination would be to have it specifically turned on for "Orphan" modules, even if you're currently choosing to turn it on globally as well. There's no real overhead for the repetition

Reply:
For example, it's turned on in 
```
{-# OPTIONS_GHC -Wno-orphans #-}
module Ouroboros.Consensus.Cardano.CanHardFork (
```

Reply:
It's also turned on in `Test.Util.Orphans.Arbitrary`:

```
{-# OPTIONS_GHC -Wno-orphans #-}

module Test.Util.Orphans.Arbitrary
```

So now I'm a bit confused...

Reply:
I only meant `-fno-warn-orphans` vs. `-Wno-orphans`.